superset/assets/javascripts/explorev2/components/ControlHeader.jsx

```diff
@@ -64,6 +65,7 @@ export default function ControlHeader({
               {' '}
             </span>
           }
+          <span>{leftNode}</span>
```

Comment:
It's hard to allow for checkbox on a single row without this. The checkbox was way far on the right when using  rightNode....